frontend/fingertips-frontend/components/forms/SearchForm/index.tsx

```diff
@@ -16,10 +16,9 @@ export const SearchForm = ({ indicator }: { indicator: string }) => {
     message: null,
     errors: {},
   };
+  // eslint-disable-next-line @typescript-eslint/no-unused-vars
   const [state, formAction] = useActionState(searchIndicator, initialState);
```

Comment:
This is because `state` isn't used, right? I assume we're getting something from `useActionState` above just calling `searchIndicator` directly then?

Could we do this instead of disabling the eslint rule entirely?
```suggestion
  const [_state, formAction] = useActionState(searchIndicator, initialState);
```
We may need to set the [varsignorepattern](https://eslint.org/docs/latest/rules/no-unused-vars#varsignorepattern) as specified [here](https://typescript-eslint.io/rules/no-unused-vars/#benefits-over-typescript).

Reply:
Nice that's a good suggestion 👍